.github/workflows/lxd-image.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build LXD image
+on:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
It is recommended to make a cleanup before the checkout 
    - name: Cleanup job workspace
      id: cleanup-job-workspace
      run: |
          rm -rf "${{ github.workspace }}"
          mkdir "${{ github.workspace }}"